test.py

```diff
@@ -53,14 +53,6 @@ def test_manager(self):
         self.assertEqual(r, "jaja-lol-a")
 
         txt = 'jaja---lol-méméméoo--a'
-        r = slugify(txt, max_length=19, word_boundary=True)
```

Comment:
Thanks. Done. The request consist of bug fix. I add the additional parameter in the next request.